susemanager/src/mgr_sync/product.py

```diff
@@ -39,6 +40,21 @@ def __init__(self, data):
     def __repr__(self):
         return self.to_ascii_row()
 
+    def __eq__(self, other):
+        return (self.friendly_name == other.friendly_name)
+
+    def __ne__(self, other):
+        return not (self == other)
+
+    def __lt__(self, other):
+        selfIsSUSE = self.friendly_name.startswith("SUSE")
+        otherIsSUSE = other.friendly_name.startswith("SUSE")
+        if selfIsSUSE and not otherIsSUSE:
+            return False
+        if not selfIsSUSE and otherIsSUSE:
```

Comment:
[nitpick]: I would prefer snake_case to be consistent with Python's style:
```suggestion
        self_is_SUSE = self.friendly_name.startswith("SUSE")
        other_is_SUSE = other.friendly_name.startswith("SUSE")
        if self_is_SUSE and not other_is_SUSE:
            return False
        if not self_is_SUSE and other_is_SUSE:
```